man/systemd.exec.xml

```diff
@@ -738,6 +738,37 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>LogLevelMax=</varname></term>
+        <listitem><para>Configures filtering by log level of log messages generated by this unit. Takes a
+        <command>syslog</command> log level, one of <option>emerg</option> (lowest), <option>alert</option>,
```

Comment:
Hmm, I know where "lowest" comes from, but it's mighty confusing to people who are not programmers familiar with the brain-dead syslog design. We (or at least I) tried to replace "log level" by "log priority" in user-facing areas.  Maybe replace "(lowest)" and "(highest)" with "(lowest log level, only highest priority messages)" and "(highest log level, also lowest priority messages)" respectively?